releasenotes/notes/improve-decomposition-controlled-one-qubit-unitaries-3ae333a106274b79.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features_circuits:
+  - |
+    Reduce the number of two-qubit gate counts when decomposing some multi-controlled
+    single-qubit unitary gates.
```

Comment:
```suggestion
    Reduce the number of two-qubit gates when decomposing multi-controlled
    single-qubit unitary gates.
```

Reply:
done in [4823321](https://github.com/Qiskit/qiskit/pull/13801/commits/48233216b4af1649159b4dbf51d549e99be54e8f)